src/frontend/deployment_details.go

```diff
@@ -0,0 +1,67 @@
+package main
+
+import (
+	"net/http"
+	"os"
+	"sync"
+
+	"cloud.google.com/go/compute/metadata"
+	"github.com/sirupsen/logrus"
+)
+
+const (
+	DetailsAreNotLoaded = 0
+	DetailsAreLoading   = 1
+	DetailsAreLoaded    = 2
+)
```

Comment:
I was inspired to do enum this way by [this article, Using Enums (and Enum Types) in Golang
](https://www.sohamkamani.com/golang/enums/).